src/System.Buffers/src/System.Buffers.csproj

```diff
@@ -1,17 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.1;netstandard2.0;net461</TargetFrameworks>
```

Comment:
Why did you decide to remove 1.1?

Reply:
We stopped building against netstandard1.x years ago (in nearly all repos): https://github.com/dotnet/announcements/issues/190